src/dave_data/datapool/hotmaps/hotmaps_request.py

```diff
@@ -0,0 +1,337 @@
+import os
+import shutil
+from pathlib import Path
+
+import geopandas as gpd
+import rasterio
+import requests
+from pyproj import CRS
+from rasterio.features import shapes
+from rasterio.mask import mask
+from shapely import box
+from shapely.geometry import shape
+
+from dave_data.datapool.hotmaps.exception.hotmaps_exceptions import InvalidBbox
+from dave_data.datapool.hotmaps.exception.hotmaps_exceptions import InvalidEPSG
+from dave_data.datapool.hotmaps.exception.hotmaps_exceptions import OutOfBbox
+
+
+def get_hotmaps_layer_info(layer_name):
+    #     This function returns a dictionary with the information about hotmaps layers
+    layer_info = {
```

Comment:
Is there an option to request this meta data also from hotmaps than hard coding it?